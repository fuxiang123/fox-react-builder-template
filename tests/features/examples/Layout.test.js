import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Layout from "../../../src/features/examples/Layout";

it("examples/Layout snapshot", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <Layout />
    </BrowserRouter>
  );
  // 快照
  expect(asFragment()).toMatchSnapshot();
});
