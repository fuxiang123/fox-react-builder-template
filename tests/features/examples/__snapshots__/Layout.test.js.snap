// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples/Layout snapshot 1`] = `
<DocumentFragment>
  <div
    class="example-page"
  >
    <header
      class="header"
    >
      <a
        class="header-bar"
        href="/"
      >
        <img
          alt="React Logo"
          src=""
        />
        <div>
          Fox-Builder
        </div>
      </a>
    </header>
    <div
      class="examples-container"
    >
      <nav
        class="side-panel"
      >
        <ul>
          <li>
            <a
              href="/examples/todolist"
            >
              示例：待办列表
            </a>
          </li>
          <li>
            <a
              href="/examples/RedditList"
            >
              示例：数据请求
            </a>
          </li>
          <li>
            <a
              href="/"
            >
              返回首页
            </a>
          </li>
        </ul>
      </nav>
      <main />
    </div>
  </div>
</DocumentFragment>
`;
